packages/glibc-dbg/PKGBUILD

```diff
@@ -0,0 +1,240 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+# Maintainer: Giancarlo Razzolini <grazzolini@archlinux.org>
```

Comment:
I've pointed out the original source of most files except some files seem to be generated or can break when adding something on it. I've added modification comments such as `BlackArch Start` `BlackArch End` or `BlackArch` since I've just done little changes like adding debug options.